### PR TITLE
Feature: script to deploy calico into the cluster

### PR DESCRIPTION
**Pull request by Ayukha**
**Why do we need it?**
To limit the out going request calls from the cluster

**Changes Proposed:**
Calico is being deployed into the cluster to provide networking and network policy.
Deploying AWS container-insights

### CODE REVIEW

File: docker/kubernetes.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+sudo apt install -y awscli  # Install the AWS CLI.
+aws --version               # Confirm the AWS CLI was installed.
+
+aws configure set a us-east-1
+aws configure set aws_access-key.id
+aws configure set aws_secret_access_key.key
+
+#install iam-authenticator
+curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.17.7/2020-07-08/bin/linux/amd64/aws-iam-authenticator
```
Comment (RishabhJain2018) [inline docker/kubernetes.sh:10]: Why do we need aws-iam-authenticator when we have the config file?

Comment (Ayukha) [reply docker/kubernetes.sh:10]: @RishabhJain2018 Config uses `get token` command from `iam-authenticator`

File: docker/kubernetes.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```
Comment (RishabhJain2018) [inline docker/kubernetes.sh:1]: Can we please keep this script in the worker folder itself. Basically at --
`scripts/workers/code_upload_worker_utils/deploy_calico.sh`

**Comment by RishabhJain2018** [comment]
Also, @Ayukha Can you please update the PR description here.

**Pull request merged**